Facebook filter

Aztekz opened the pull request:
Up to that point you were on board

Commit:
Update specific.txt

Commit:
Update specific.txt

Change to one-line-param.

Aztekz commented:
Will this not be merged or is there something wrong with it? Should i close this?

Alex-302 commented:
As I know, removing of `spm` breaks important featulres on Aliexpress. If these sites has same developers/owners, they also may be broken.

Alex-302 commented on TrackParamFilter/sections/specific.txt:
```diff
@@ -220,3 +220,7 @@
 ||websearch.rakuten.co.jp/Web?$removeparam=ref
 ||rakuten.co.jp^*&trflg=$removeparam=trflg
 ||search.yahoo.co.jp^$removeparam=fr
+! Daraz & Shop (Samesite)
```
`||*.` is redundant. Better `||daraz.$......`

Aztekz replied:
Fixed.

Commit:
Update specific.txt

Remove Subdomain redundancies.

Aztekz commented:
> As I know, removing of `spm` breaks important featulres on Aliexpress. If these sites has same developers/owners, they also may be broken.

Is there any specific case where removing this have caused issues? (I'll look into it)

I have been browsing these site's without any issues so far though, so I can't really say much about Aliexpress/Alibaba (Haven't tested them out).

Alex-302 commented:
@Aztekz Problems, when used profile and payments.
Also please provide sample values for each parameter.

Aztekz commented:
> @Aztekz Problems, when used profile and payments.
> Also please provide sample values for each parameter.

I am doing some additional tests to see if any problems occur. Will provide samples (Do note, cerrtain 'params' are not always available/visible).

Commit:
Update specific.txt

Fixed an issue where it would remove `param` and redirect to main page if the words ended/started matched some of params like `c`, `mp`, `spm` etc. It is more "case sensitive" now.

Aztekz commented:
@Alex-302 I made a purchase to check if any issue exists (no issue so far), viewed all links in profile (no issues so far [ordering/sorting/reviewing etc]), also did some more testing (fixed an issue with "case" detection & added 2 new param & fixed an old param) Here are some samples from all the sites (_Apologies for the wall of text_):


**`PK`**

> **_Before_**
> > - https://www.daraz.pk/wow/gcp/daraz/daily/pk/fashion-clearance-sale/homepage?spm=a2a0e.home.top.dbr1.35e34937eIrOff&scm=1003.4.icms-zebra-5029921-2827790.OTHER_6046165446_7440302
> > - https://www.daraz.pk/products/best-attractive-tile-stickers-pack-of-12-pieces-for-home-decor-walls-self-adhesive-designs-48-inch-x-48-inch-azamsons-i220263789-s1437765861.html?spm=a2a0e.home.flashSale.7.35e34937NwumYN&search=1&mp=1&c=fs
> > - https://pages.daraz.pk/wow/i/pk/landingpage/lp-collection-product?spm=a2a0e.home.collections.8.35e34937y0QFfk&wh_weex=true&abtest=&pos=7&abbucket=&clickTrackInfo=1007.19892.190918.100200300000000_1d5ed394-75ee-4b01-ac26-084dc112fc93_0__pk_2577335_6%3A0_147924152_9_-1.0_-100_0__1_154174048_9_0.0_-100_0__2_101325224_9_0.0_-100_0&themeID=2577335&acm=icms-zebra-5029921-2990825.1003.1.2640676&algArgs=2577335_147924152%3A10000433%3A10000336%3A39704-154174048%3A10000741%3A10000336%3A39704-101325224%3A10000428%3A10000336%3A39704&scm=1007.19892.190918.100200300000000
> 
> **_After_**
> > -  https://www.daraz.pk/wow/gcp/daraz/daily/pk/fashion-clearance-sale/homepage
> > - https://www.daraz.pk/products/best-attractive-tile-stickers-pack-of-12-pieces-for-home-decor-walls-self-adhesive-designs-48-inch-x-48-inch-azamsons-i220263789-s1437765861.html
> > - https://pages.daraz.pk/wow/i/pk/landingpage/lp-collection-product?wh_weex=true (Removing last param `wh_weex` will cause issues)


**`BD`**

> **_Before_**
> > - https://pages.daraz.com.bd/wow/gcp/daraz/channel/bd/covid-essentials/protection-from-covid?scm=1003.4.icms-zebra-100031732-2893410.OTHER_6045878767_7432298
> > - https://pages.daraz.com.bd/wow/i/bd/landingpage/lp-collection-product?spm=a2a0e.home.collections.7.735212f7V8Fsit&wh_weex=true&abtest=&pos=6&abbucket=&clickTrackInfo=1007.19892.190918.100200300000000_1daf88f7-cce3-4045-b98e-c7ce2def5e19_0__bd_12313_5%3A0_136584392_9_-1.0_-100_0__1_157252615_9_0.0_-100_0__2_157248583_9_0.0_-100_0&themeID=12313&acm=icms-zebra-100031732-2895790.1003.1.2560461&algArgs=12313_136584392%3A9009%3A8706%3A39704-157252615%3A9064%3A8706%3A39704-157248583%3A9064%3A8706%3A39704&scm=1007.19892.190918.100200300000000
> > - https://www.daraz.com.bd/household-supplies/?spm=a2a0e.home.categories.10.4c2e12f7eO5B1t&abtest=&item_id=157496393&pos=10&abbucket=&clickTrackInfo=0e004f97-1561-49d5-a6c4-c8abe5a5ca45__10000535__157496393__static__0.03__158061__7253&up_id=157496393&from=hp_categories&acm=icms-zebra-100031732-2891980.1003.1.2542290&scm=1007.17253.158061.0&version=v2
> 
> **_After_**
> > - https://pages.daraz.com.bd/wow/gcp/daraz/channel/bd/covid-essentials/protection-from-covid
> > - https://pages.daraz.com.bd/wow/i/bd/landingpage/lp-collection-product?wh_weex=true
> > - https://www.daraz.com.bd/household-supplies/


**`LK`**

> **_Before_**

> > - https://pages.daraz.lk/wow/gcp/daraz/megascenario/lk/mobile-week-21/live?spm=a2a0e.home.campaignbanner.1.675a4625qgNFEb&scm=1003.4.icms-zebra-100031632-4035077.OTHER_6046040749_7437277
> > - https://pages.daraz.lk/wow/i/lk/landingpage/lp-collection-product?spm=a2a0e.home.collections.3.675a4625qgNFEb&wh_weex=true&abtest=&pos=2&abbucket=&clickTrackInfo=1007.19892.190918.100200300000000_2be84920-894e-4540-a71d-6f86a51844a3_0__lk_1863414_1%3A0_101936285_9_-1.0_-100_0__1_112254121_9_0.0_-100_0__2_112554226_9_0.0_-100_0&themeID=1863414&acm=icms-zebra-100031632-2911261.1003.1.2560665&algArgs=1863414_101936285%3A7234%3A3752%3A117392-112254121%3A10002138%3A3752%3A123504026-112554226%3A7232%3A3752%3A64537&scm=1007.19892.190918.100200300000000
> > - https://www.daraz.lk/products/stereo-extra-bass-high-quality-headphones-xb450-xb550-earphones-i119938505-s1038958948.html?spm=a2a0e.home.just4u.22.173f4625j6agMN&scm=1007.28811.237375.0&pvid=7b7d96e9-55df-49f5-a8b0-e4bc511cdc6d&clickTrackInfo=pvid%3A7b7d96e9-55df-49f5-a8b0-e4bc511cdc6d%3Bchannel_id%3A0000%3Bmt%3Ahot%3Bitem_id%3A119938505%3B
> 
> **_After_**
> > - https://pages.daraz.lk/wow/gcp/daraz/megascenario/lk/mobile-week-21/live
> > - https://pages.daraz.lk/wow/i/lk/landingpage/lp-collection-product?wh_weex=true
> > - https://www.daraz.lk/products/stereo-extra-bass-high-quality-headphones-xb450-xb550-earphones-i119938505-s1038958948.html

**`NP`**

> **_Before_**
> > - https://www.daraz.com.np/?spm=a2a0e.pdp.venture.5.4b694423C0ka6e&scm=1003.4.icms-zebra-100024412-2878770.OTHER_5487142937_2528371
> > - https://www.daraz.com.np/shop/goldstar-nepal/?spm=a2a0e.11779170.campaignbanner.1.71f82d2bwOvqcI&scm=1003.4.icms-zebra-100031662-3734717.OTHER_6046145144_7439811
> > - https://www.daraz.com.np/shop/sherpa-outdoor?spm=a2a0e.11779170.officialStores.5.395d2d2bjWmEIo&abtest=&pos=4&abbucket=&up_id=104066779,104192188&from=hp_officialstore&acm=icms-zebra-100031662-2991806.1003.1.2640706&scm=1007.19895.182511.100200300000000
> 
> **_After_**
> > - https://www.daraz.com.np/
> > - https://www.daraz.com.np/shop/goldstar-nepal/
> > - https://www.daraz.com.np/shop/sherpa-outdoor

**`MM`**

> **_Before_**
> > - https://pages.shop.com.mm/wow/gcp/daraz/channel/mm/shop_mart_mm/Shop-Mart-MM-MCP-update?spm=a2a0e.home.feature_nav.2.672a7fe4Dx9WRR&scm=1003.4.icms-zebra-100031612-2974982.OTHER_6046138473_7439040
> > - https://pages.shop.com.mm/wow/gcp/daraz/megascenario/mm/techweek/gc_techweek_main_page?spm=a2a0e.home.feature_nav.3.672a7fe4Dx9WRR&scm=1003.4.icms-zebra-100031612-2974982.OTHER_6046138474_7439040
> > - https://pages.shop.com.mm/wow/gcp/daraz/megascenario/mm/techweek/mobile-accessories?spm=a2a0e.home.feature_nav.4.672a7fe4Dx9WRR&scm=1003.4.icms-zebra-100031612-2974982.OTHER_6046138475_7439040
> 
> **_After_**
> > - https://pages.shop.com.mm/wow/gcp/daraz/channel/mm/shop_mart_mm/Shop-Mart-MM-MCP-update
> > - https://pages.shop.com.mm/wow/gcp/daraz/megascenario/mm/techweek/gc_techweek_main_page
> > - https://pages.shop.com.mm/wow/gcp/daraz/megascenario/mm/techweek/mobile-accessories)

Commit:
Update specific.txt

fixed `sugg` param, it is rarely seen when performing search.

Alex-302 commented:
@Aztekz Removed 
`||shop.*$removeparam=/spm=|scm=|from=|keyori=|sugg=|search=|mp=|c=|^abtest|^abbucket|pos=|themeID=|algArgs=|clickTrackInfo=|acm=|item_id=|version=|up_id=|pvid=/`
It breaks https://github.com/AdguardTeam/AdguardFilters/issues/90537

Aztekz commented:
@Alex-302 Apologies for the late reply. This doesn't seem to be effected anymore? (Issue #90537) I don't know what caused the issue in the first place as I cannot seem to find anything with the ***.shop** domain/subdomain in that site.
